public/app/features/alerting/unified/components/bridges/DeclareIncidentButton.tsx

```diff
@@ -12,7 +12,7 @@ interface Props {
   url?: string;
 }
 
-export const DeclareIncident: FC<Props> = ({ title = '', severity = '', url = '' }) => {
+export const DeclareIncidentButton: FC<Props> = ({ title = '', severity = '', url = '' }) => {
```

Comment:
💯

Reply:
💯 about this name change

Reply:
I'm pretty sure the component has been around since before we made changes to the style guide but I can pick this up in this PR.